plugins/BEdita/Core/src/ORM/QueryFilterTrait.php

```diff
@@ -90,11 +90,13 @@ public function fieldsFilter(Query $query, array $options)
                         $in[] = $value;
                         continue;
                     }
+
                     $exp = $this->operatorExpression($exp, $operator, $field, $value);
                 }
-                if (!empty($in)) {
-                    $exp = $exp->in($field, $in);
-                }
+            }
+
+            if (!empty($in)) {
+                $exp = $exp->in($field, $in);
```

Comment:
Careful. This should be restored where it used to be, inside `foreach ($options as $field => $conditions)`.